### PR TITLE
Changed documentation for incorrect parameters

### PR DESCRIPTION
**Pull request by niyatiagg**
### Description
Changed the description of search.cancel_after_time_interval and search.default_search_timeout parameters.

### Issues Resolved
Closes [#6843]

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _install-and-configure/configuring-opensearch/search-settings.md [hunk 1]
```diff
@@ -17,9 +17,9 @@ OpenSearch supports the following search settings:
 
 - `search.default_allow_partial_results` (Dynamic, Boolean):  A cluster-level setting that allows returning partial search results if a request times out or a shard fails. If a search request contains an `allow_partial_search_results` parameter, the parameter takes precedence over this setting. Default is `true`. 
 
-- `search.cancel_after_time_interval` (Dynamic, time unit): A cluster-level setting that specifies the maximum amount of time that a search request can run before it is canceled at the shard level. After this time has been reached, a request is stopped and all associated tasks are canceled. Default is `-1`.
+- `search.cancel_after_time_interval` (Dynamic, time unit): A cluster-level setting that sets the default timeout for all search requests at the coordinating node level. If the `timeout` is specified in the search request, it takes precedence over this setting. Default is `-1` (no timeout).
```
Comment (Naarcha-AWS) [inline _install-and-configure/configuring-opensearch/search-settings.md:20]:
```suggestion
- `search.cancel_after_time_interval` (Dynamic, time unit): A cluster-level setting that sets the default timeout for all search requests at the coordinating0node level. If the `timeout` interval is specified in the search request, that interval takes precedent over the configured setting. Default is `-1` (no timeout).
```

File: _install-and-configure/configuring-opensearch/search-settings.md [hunk 2]
```diff
@@ -17,9 +17,9 @@ OpenSearch supports the following search settings:
...
 
-- `search.default_search_timeout` (Dynamic, time unit): A cluster-level setting that sets the default timeout for all search requests at the coordinating node level. If the `timeout` is specified in the search request, it takes precedence over this setting. Default is `-1` (no timeout).
+- `search.default_search_timeout` (Dynamic, time unit): A cluster-level setting that specifies the maximum amount of time that a search request can run before it is canceled at the shard level. After this time has been reached, a request is stopped and all associated tasks are canceled. Default is `-1`.
```
Comment (Naarcha-AWS) [inline _install-and-configure/configuring-opensearch/search-settings.md:22]:
```suggestion
- `search.default_search_timeout` (Dynamic, time unit): A cluster-level setting that specifies the maximum amount of time that a search request can run before the request is canceled at the shard-level. After the specified time has been reached, the request is stopped and all associated tasks are canceled. Default is `-1`.
```

**Review by Naarcha-AWS — CHANGES_REQUESTED** [review]
I'm slightly confused as the why `timeout` is specified in the setting that says `after_time_interval` versus the `search_timeout` setting? Can you provide some additional context here?

**Comment by jainankitk** [comment]
> I'm slightly confused as the why timeout is specified in the setting that says after_time_interval versus the search_timeout setting? Can you provide some additional context here?

The `search.default_search_timeout` setting was originally present, but it only timed out for shard level tasks. We wanted to achieve the same at request level, but did not want to change the behavior of existing setting. Hence, introduced new setting for this. This issue has more context - https://github.com/opensearch-project/OpenSearch#817

File: _install-and-configure/configuring-opensearch/search-settings.md [hunk 3]
```diff
@@ -17,9 +17,9 @@ OpenSearch supports the following search settings:
...
+- `search.cancel_after_time_interval` (Dynamic, time unit): A cluster-level setting that sets the default timeout for all search requests at the coordinating node level. If the `timeout` interval is specified in the search request, that interval takes precedent over the configured setting. Default is `-1` (no timeout).
 
-- `search.default_search_timeout` (Dynamic, time unit): A cluster-level setting that sets the default timeout for all search requests at the coordinating node level. If the `timeout` is specified in the search request, it takes precedence over this setting. Default is `-1` (no timeout).
+- `search.default_search_timeout` (Dynamic, time unit): A cluster-level setting that specifies the maximum amount of time that a search request can run before the request is canceled at the shard-level. After the specified time has been reached, the request is stopped and all associated tasks are canceled. Default is `-1`.
```
Comment (jainankitk) [inline _install-and-configure/configuring-opensearch/search-settings.md:22]: Can we remove the text `After the specified time has been reached, the request is stopped and all associated tasks are canceled.` from here and add to documentation of `search.cancel_after_time_interval`?

**Review by jainankitk — APPROVED** [review]

**Review by Naarcha-AWS — APPROVED** [review]

**Pull request merged**